perf/small_alloc_perf.c

```diff
@@ -333,6 +515,26 @@ int main(int argc, char* argv[])
 	if (argc == 2 && !strcmp(argv[1], "-h")) //human clear output
 		human = true;
 
+	for (int i = 1; i < argc; i++) {
+		if (!strcmp(argv[i], "-h")) {
```

Comment:
-h as a rule means --help and provides information about available launch keys and assists user in launching program..
I suggest to rework -h flag. For changing results format we can use another flag.